packages/schematics/angular/application/index.ts

```diff
@@ -307,6 +307,7 @@ export default function (options: ApplicationOptions): Rule {
     }
     validateProjectName(options.name);
     options.prefix = options.prefix || 'app';
+    options.style = options.style || Style.Css;
```

Comment:
Is this need? Shouldn't it get the default which is set in the schema?

Reply:
I think it is needed.
If we call `ng generate application` directly it gets the default value. But it doesn't get it when called via `ng new`. Maybe there is a bug somewhere else.

I think this is why we also have `options.prefix = options.prefix || 'app';`

Reply:
It technically shouldn't be needed but currently it might.
Outside of this PR, this should be investigated, fixed if it is indeed broken, and those two lines removed.

Reply:
Yes. Should this PR be on hold until the other issue is fixed?

Reply:
I don't think so.  The line above it is already there.  Can you add a comment explaining the situation?

Reply:
Added a comment. Let me know if something else needs to be changed/added.